styles/linter-ui.less

```diff
@@ -17,64 +17,85 @@
   padding: 5px 5px 2px 5px;
   font-family: @font-family;
 
-  .message-with-severity(@color) {
+  .linter-message {
+    padding: 10px;
+
     color: #fff - @inset-panel-background-color; // invert;
     background: @app-background-color;
 
-    border-left: 8px solid @color;
-    border-right: 1px solid fade(contrast(@inset-panel-background-color), 5%);
-    &:first-child {
-      border-top: 1px solid fade(contrast(@inset-panel-background-color), 5%);
-    }
-    &:last-child {
-      border-bottom: 1px solid fade(contrast(@inset-panel-background-color), 5%);
-    }
+    .linter-excerpt {
+      padding: 5px;
 
-    &:last-child::before {
-      border-color: transparent transparent @color @color;
-    }
+      // border between messages
+      &:not(:last-child) {
+        border-bottom: 1px solid fade(contrast(@inset-panel-background-color), 5%);
+      }
 
-    a {
-      color: @text-color;
-    }
-  }
+      &.error {
+        .message-with-severity(@text-color-error);
+      }
+      &.warning {
+        .message-with-severity(@text-color-warning);
+      }
+      &.info {
+        .message-with-severity(@text-color-info);
+      }
 
-  .linter-message {
-    &.error {
-      .message-with-severity(@text-color-error);
-      &::after {
-        color: @text-color-error;
+      // message severity function
+      .message-with-severity(@color) {
+        border-left: 4px solid @color;
+
+        &:last-child::before {
+          border-color: transparent transparent @color @color;
+        }
       }
-    }
-    &.warning {
-      .message-with-severity(@text-color-warning);
-      &::after {
-        color: @text-color-warning;
+      .provider-name {
+        display: inline;
+        font-weight: bolder;
       }
-    }
-    &.info {
-      .message-with-severity(@text-color-info);
-      &::after {
-        opacity: 0;
+
+      // align button with message
+      .linter-buttons-right {
+        margin-top: 5px; // TODO align
```

Comment:
CSS fights with me when I want to place the URL button in the middle.

Reply:
flexbox align-items?

Reply:
I tried this, but it screws up the provider name. 
What I Want is:
- provider-name and linter text to be like a normal continuous paragraph, but provider-name be bolder. These too should be one group together.

- align all the items in the center

Reply:
> align all the items (including the linter-text group) in the center

centered vertically correct?

Reply:
Yes

Reply:
This is the only thing holding this PR back. Should make this a follow-up PR? There are CSS experts out there that can probably fix the issue in other PRs.

Reply:
![image](https://user-images.githubusercontent.com/97994/106544477-38443700-64cd-11eb-97eb-e5d239f118e2.png)

is this what you are going for?

Reply:
Yes:
 - The right buttons should be placed at the far end of the datatip
 - In the long datatips, the provider name should be in the same paragraph as the text:
![image](https://user-images.githubusercontent.com/16418197/106408052-301fc500-6403-11eb-8bfa-7d6a16ad3084.png)